docs/parameters-composite.md

```diff
@@ -1,40 +1,43 @@
 # Standard composite service (example)
 
-Parameters to deploy a standard wordpress service, including the database and the s3proxy service as a ["sidecar"](https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/fargate-security-considerations.html)
+Parameters to deploy a standard wordpress service, including the database and the s3proxy service as a ["sidecar"](https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/fargate-security-considerations.html).
+NOTE: mod_shib has been removed from the underlying docker image and replaced with reduced responsibility that delegates authentication to a lambda@edge origin request function with its cloudfront distribution. The docker image upon which the main wordpress container is based has these changes reflected in its baseline image, and the second stage portion of the build is based on a manifest .ini file that modifies those bu-specific php customizations dealing with authentication to accomodate this. *(example: `MANIFEST_INI_FILE=wp-manifests/devl/jaydub-bulb.ini`)*
 
 *Note that `WORDPRESS.env.s3ProxyHost` is removed entirely, which means the necessary default to `"localhost"`* for sidecar reference.
 
 ```
 {
   "SCENARIO": "composite",
```

Comment:
I'm just wondering if maybe it's enough to refer to the _example-context.json here instead of putting the entire context in the .md file?